core/src/libraries/BalanceSheet.sol

```diff
@@ -85,10 +85,10 @@ library BalanceSheet {
         uint256 iv,
         uint256 nSigma,
         uint56 metric
-    ) internal pure returns (uint160 a, uint160 b, bool isSus) {
+    ) internal pure returns (uint160 a, uint160 b, bool seemsLegit) {
         unchecked {
             iv = SoladyMath.clamp((nSigma * iv) / 10, PROBE_PERCENT_MIN, PROBE_PERCENT_MAX);
-            isSus = metric > _manipulationThreshold(_effectiveCollateralFactor(iv));
+            seemsLegit = metric < _manipulationThreshold(_effectiveCollateralFactor(iv));
```

Comment:
Should this be `<=`?

Reply:
Nah it doesn't really matter. The manipulation stuff isn't an exact science. This way's more conservative and saves a few gas.